### DISCUSSION TITLE
Can I make an app on top of this project and publish it to Play Store and App Store?

### DISCUSSION THREAD

**Discussion by Ohpleaseman** [organicmaps/organicmaps#8342] [Q&A] [answered]
If yes, how do you publish it? If no can I use particular code from project on my other projects?

**Comment by biodranik** [comment, marked as answer]
Please check the terms of the Apache2 license and notice in the repository. What is your goal? What kind of app do you want to publish?

**Reply by Ohpleaseman** [reply]
We want to make a tourism app that would actually be ONLY about Tajikistan.
